imports/plugins/core/orders/client/components/OrderHeader.js

```diff
@@ -72,7 +66,7 @@ function OrderHeader(props) {
           {paymentStatusChip}
           <Grid item>
             <Button
-              href={handleClickPrintLink()}
+              onClick={() => Reaction.Router.go(`/operator/orders/print/${order.referenceId}`)}
```

Comment:
Either use the `href`, or the react-router `Link`. `Reaction.Router.go` is old router